bundles/org.openhab.binding.visualcrossing/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,1975 @@
+	<channel-type id="precip-type-channel">
+		<item-type>String</item-type>
+		<label>Precip Type</label>
+		<description>
+			An comma separated array indicating the type(s) of precipitation expected or that occurred. Possible
+			values include `rain`, `snow`, `freezingrain` and `ice`.
+		</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="precip-cover-channel">
+		<item-type unitHint="%">Number</item-type>
+		<label>Precip Cover</label>
+		<description>The proportion of hours where there was non-zero precipitation</description>
+		<state readOnly="true" min="0" max="100" pattern="%.0f %unit%"/>
+	</channel-type>
+	<channel-type id="snow-channel">
+		<item-type>Number:Length</item-type>
+		<label>Snow</label>
+		<description>The amount of snow that fell or is predicted to fall</description>
+		<category>Snow</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="snow-depth-channel">
+		<item-type>Number:Length</item-type>
+		<label>Snow</label>
+		<category>Snow</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="visibility-channel">
+		<item-type>Number:Length</item-type>
+		<label>Visibility</label>
+		<description>Distance at which distant objects are visible</description>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="cloud-cover-channel">
+		<item-type unitHint="%">Number:Dimensionless</item-type>
+		<label>Cloud Cover</label>
+		<description>How much of the sky is covered in cloud ranging from 0–100%</description>
+		<category>Sun_Clouds</category>
+		<state readOnly="true" min="0" max="100" pattern="%.0f %unit%"/>
+	</channel-type>
+	<channel-type id="solar-radiation-channel">
+		<item-type>Number:Intensity</item-type>
+		<label>Solar Radiation</label>
+		<description>The solar radiation power at the instantaneous moment of the observation (or forecast prediction).
+			See the
+			full solar radiation data documentation
+			(https://www.visualcrossing.com/resources/documentation/weather-data/how-to-obtain-solar-radiation-data/)
+			and Wind and
+			Solar Energy
+			(https://www.visualcrossing.com/resources/documentation/weather-api/energy-elements-in-the-timeline-weather-api/)
+			pages.
+		</description>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="solar-energy-channel">
+		<item-type>Number</item-type>
+		<label>Solar Energy</label>
+		<description>(MJ /m^2) Indicates the total energy from the sun that builds up over an hour or day. See the full
+			solar
+			radiation data documentation
+			(https://www.visualcrossing.com/resources/documentation/weather-data/how-to-obtain-solar-radiation-data/)
+			and Wind and
+			Solar Energy
+			(https://www.visualcrossing.com/resources/documentation/weather-api/energy-elements-in-the-timeline-weather-api/)
+			pages.
+		</description>
+		<state readOnly="true" pattern="%.2f MJ/m²"/>
```

Comment:
@lsiepel  is this correct?

Reply:
I guess so. The unit is hard set because UoM does not support this unit, so no conversion can be performed.